_posts/ref/2022-01-18-wire-v1.md

```diff
@@ -0,0 +1,208 @@
+---
+layout: page
+title:  "WIRE-V1 REST API and MQTT Request Handler"
+categories: [ref]
+---
+
+The WIRE-V1 cloud request handler and REST APIs allow to perform update, delete and get operations on the current Wire Graph. This request handler provides the equivalent of the `org.eclipse.kura.wire.graph.WireGraphService` Java API.
```

Comment:
> The WIRE-V1 cloud request handler and REST APIs allow to perform update, delete and get operations on the current Wire Graph.

The WIRE-V1 cloud request handler and the corresponding REST APIs allow to update, delete and get the current Wire Graph status.